tests/datalab/issue_manager/conftest.py

```diff
@@ -0,0 +1,52 @@
+from hypothesis import strategies as st
+import numpy as np
+from scipy.sparse import csr_matrix
+
+
+@st.composite
+def knn_graph_strategy(draw, num_samples, k_neighbors):
```

Comment:
would be helpful to have a high level docstring for this function, describing what properties of the KNN graph you would expect from this function

Reply:
Also the naming of this function is a bit confusing, could we call it something like `create_knn_graph_with_strategy`?